modules/browser_object_navigation.py

```diff
@@ -88,6 +86,11 @@ def get_awesome_bar(self) -> Page:
         self.awesome_bar = self.driver.find_element(*self._awesome_bar)
         return self
 
+    def clear_awesome_bar(self) -> Page:
+        self.get_awesome_bar()
+        self.awesome_bar.clear()
```

Comment:
This looks a little weird, you're not returning the awesome bar from `get_awesome_bar` but you can clear it?